plugins/code_eval.rb

```diff
@@ -37,4 +37,10 @@ def code_eval(m, lang, code)
   rescue EvalIn::ConnectionError
     m.reply "Failure while connecting to evaluation service.", true
   end
+
+  match /eval$/, method: :list_languages, strip_colors: true
+
+  def list_languages(m)
+    m.reply "Known language: #{EvalIn::Result::LANGS.keys * ", "}", true
```

Comment:
Something like this would be a little more readable:

```ruby
langs = EvalIn::Result::LANGS.join(", ")
m.reply "Known languages: #{langs}", true
```

Reply:
will change in the later evening, thx :)